libcnb-package/src/cross_compile.rs

```diff
@@ -59,6 +59,31 @@ The easiest way to install 'musl-gcc' is to install the 'musl-tools' package:
 - https://packages.debian.org/bullseye/musl-tools"#,
             )),
         }
+    } else if target_triple.as_ref() == AARCH64_UNKNOWN_LINUX_MUSL && cfg!(target_os = "linux") {
+        let gcc_binary_path = "aarch64-linux-gnu-gcc";
```

Comment:
@hone Was there a reason this is `aarch64-linux-gnu-gcc` not `aarch64-linux-musl-gcc`? We're using MUSL for everything else?

Reply:
Filed #724.